src/app/ui/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -63,6 +63,7 @@ const dropdownContentStyles = css({
   boxShadow:
     '0px 12px 24px 0px rgba(18, 16, 15, 0.08), 0px 4px 8px 0px rgba(18, 16, 15, 0.08), 0px 0px 2px 0px rgba(18, 16, 15, 0.08)',
   p: '0',
+  width: '240px',
```

Comment:
We shouldn't add a fixed width here? Max-width perhaps?

Reply:
You're right, this was increasing the width of the `recipient-type-dropdown` and causing a bug. 

I tried `max-width` but that doesn't work. 

I'll figure something out to make it wider for this specific dropdown in `settings`, hopefully to auto fit the content

Reply:
I wonder if we made the styling of these shared components too strict? I know @kyranjamie had to create an `unstyled` version of the trigger. And, I had to create an `IconButton` here bc I need access to the data open state. Maybe we should export the default styles so we can add to them if needed and properly override the the styles applied to the Radix primitive?

Reply:
@pete-watters can't we add a width to the child item content of the dropdown to get the desired width?

@fbwoolf what other cases are there we need to cover that don't work with current set of styles? We could make more flexible but that comes at the cost of consistency across the app.

Reply:
Not too sure I know all use cases, but we did do this with the tooltip I noticed where we pass in an additional `className` prop to expand the default styles.

Reply:
@kyranjamie the way it's setup right now, the components only accept the props that the Radix components accept. 

I tried passing a `className` to `RadixDropdownMenu.Content` but it just overrides the original one applied so I was thinking to:
- use the [classnames addon](https://github.com/JedWatson/classnames)
- try change the classname declaration in `DropdownMenu` to use ES6 template literals to join classnames

I checked `tooltip`, thanks @fbwoolf . There we accept a separate className prop and join it to the radix style which is what we can do here